src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentVerifyRequest.cs

```diff
@@ -0,0 +1,72 @@
+﻿using SwedbankPay.Sdk.PaymentOrders;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.Payments.CardPayments
+{
+    public class CardPaymentVerifyRequest
+    {
+        public CardPaymentVerifyRequest(Operation operation,
+                              CurrencyCode currency,
+                              string description,
+                              string userAgent,
+                              CultureInfo language,
+                              Urls urls,
+                              PayeeInfo payeeInfo,
+                              string payerReference = null,
+                              bool generatePaymentToken = false,
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)